packages/expo-modules-autolinking/src/ReactImportsPatcher.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+import glob from 'fast-glob';
+import fs from 'fs-extra';
+import path from 'path';
+
+import { PatchReactImportsOptions } from './types';
+
+/**
+ * Remove all double-quoted react header imports
+ * @param dirs target directories to patch
+ * @param options PatchReactImportsOptions
+ */
+export async function patchReactImportsAsync(dirs: string[], options: PatchReactImportsOptions) {
+  const headerSet = await generateReactHeaderSetAsync(
+    path.join(options.podsRoot, 'Headers', 'Public', 'React-Core', 'React')
+  );
+  await Promise.all(dirs.map((dir) => patchDirAsync(headerSet, dir, options.dryRun)));
+}
+
+/**
+ * Generate `React-Core` public header names as a set, will transform necessary headers based on this set.
+ */
+async function generateReactHeaderSetAsync(reactHeaderDir: string): Promise<Set<string>> {
+  const result = new Set<string>();
+  const files = await glob('*.h', { cwd: reactHeaderDir });
+  for (const file of files) {
+    result.add(file);
+  }
+  return result;
+}
+
+/**
+ * Patch imports from a file
+ * @param headerSet prebuilt React-Core header set
+ * @param file target patch file
+ * @param dryRun true if not writing changes to file
+ */
+export async function patchFileAsync(headerSet: Set<string>, file: string, dryRun: boolean) {
+  let changed = false;
+  const content = await fs.readFile(file, 'utf-8');
+  const transformContent = content.replace(
+    /^#import\s+?"(.+)"$/gm,
```

Comment:
```suggestion
    /^#import\s+"(.+)"$/gm,
```
Should be fine to just use `\s+` since the regex is meant to consume all whitespace until `"`